modules/rollback-operator-upgrades-cli.adoc

```diff
@@ -10,157 +11,97 @@ ifeval::["{context}" == "upgrade-cloudsvc-operator"]
 :cloud-svc:
 endif::[]
 
-You can roll back the Operator version by using CLI commands.
+You can roll back the Operator version by using command-line interface (CLI) commands.
 
 .Procedure
 
-. Delete the {olm} subscription by running the following command:
-* For {ocp}, run the following command:
+
+. Delete the {olm-first} subscription and cluster service version (CSV):
 +
-[source,terminal]
-----
-$ oc -n rhacs-operator delete subscription rhacs-operator
-----
-* For Kubernetes, run the following command:
+[IMPORTANT]
+====
+If you use Kubernetes, enter `kubectl` instead of `oc`.
+====
+
+.. To delete the {olm} , run the following command:
 +
 [source,terminal]
 ----
-$ kubectl -n rhacs-operator delete subscription rhacs-operator
+$ oc -n rhacs-operator delete subscription rhacs-operator
 ----
-. Delete the cluster service version (CSV) by running the following command:
-* For {ocp}, run the following command:
+
+.. To delete the CSV, run the following command:
 +
 [source,terminal]
 ----
 $ oc -n rhacs-operator delete csv -l operators.coreos.com/rhacs-operator.rhacs-operator
 ----
-* For Kubernetes, run the following command:
-+
-[source,terminal]
-----
-$ kubectl -n rhacs-operator delete csv -l operators.coreos.com/rhacs-operator.rhacs-operator
-----
 ifndef::cloud-svc[]
-. Determine the previous version you want to roll back to by choosing one of the following options:
-* If the current Central instance is running, query the {product-title-short} API to get the rollback version by running the following command:
-+
-[source,terminal]
-----
-$ curl -k -s -u <user>:<password> https://<central hostname>/v1/centralhealth/upgradestatus | jq -r .upgradeStatus.forceRollbackTo
-----
-* If the current Central instance is not running, perform the following steps:
+
+. Uninstall the current {rh-rhacs-first} Operator:
```

Comment:
The instructions from lines 19 until here are also about uninstalling the operator (removing the CSV and subscription basically uninstalls the operator)

This is the equivalent procedure via the UI.